packages/cfd/src/Components/migration-success-modal/__tests__/migration-success-modal-content.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { CFDStoreProvider } from '../../../Stores/Modules/CFD/Helpers/useCfdStores';
+import MigrationSuccessModalContent from '../migration-success-modal-content';
+
+const mock_store = mockStore({
+    ui: {
+        is_mobile: false,
+    },
+});
```

Comment:
```suggestion
const mock_store = mockStore({});
```
no need to specify it, it is default value for mock store